### DISCUSSION TITLE
v13.7.1 or master not working

### DISCUSSION THREAD

**Discussion by r3ing** [w9jds/firebase-action#229] [General]
Since yesterday the master version has been giving an error, I had to use the v13.5.2 version and everything continued to work fine.

**Comment by w9jds** [comment]
This was due to a step in the action that handled automatically bumping the firebase-tools version wasn't deploying the docker image after doing so. Like stated in the README, you can specify a specific version, as just pointing to latest tends to be unstable. However, this has been fixed and the image now properly auto deploys when it bumps the version.

I would also point out that `setup-firebase` is also available which is more stable and lets you specify the firebase-tools version you would like and tends to be faster in your pipeline.
